test/errors/tupleDecls-brief.good

```diff
@@ -0,0 +1,3 @@
+error in tupleDecls.chpl:1: attempt to use tuple declaration to split a value of non-tuple type 'type UnknownType'
```

Comment:
Maybe just "cannot de-tuple an expression of non-tuple type xyz"?  It seems like most of our error messages in brief mode are more like a description of what the user has done wrong rather than a statement summarizing what they've attempted to do?

Reply:
I agree with Brad that it should be a description of what the user has done wrong (and this reminds me to do the same on https://github.com/chapel-lang/chapel/pull/20866), but I don't think "de-tuple" is a spec term or easily understood.

Reply:
I've updated this error message to be more about what cannot be done and less about what was found.